website/docs/key_generation.md

```diff
@@ -209,3 +209,8 @@ Partition path generated from key generator: "2020040118"
 
 Input field value: "20200401" <br/>
 Partition path generated from key generator: "04/01/2020"
+
+## Related Resources
```

Comment:
This video might be relevant for de-duping, does not address key generation. Lets remove this